core/src/main/resources/hudson/model/ComputerSet/sidepanel.jelly

```diff
@@ -30,7 +30,9 @@ THE SOFTWARE.
   <l:header />
   <l:side-panel>
     <l:tasks>
-      <l:task href="../configureClouds" icon="symbol-cloud" permission="${app.SYSTEM_READ}"
+      <l:task href="../computer" icon="symbol-computer"
```

Comment:
Rather than change `Functions`, this should obtain the correct URL from `Functions#getNearestAncestorUrl`.